src/job/stagingJobHandler.ts

```diff
@@ -57,6 +57,8 @@ export class StagingJobHandler extends JobHandler {
       this.currJob.buildCommands[this.currJob.buildCommands.length - 1] = 'make next-gen-html';
       if (this.currJob.payload.repoName === 'devhub-content-integration') {
         this.currJob.buildCommands[this.currJob.buildCommands.length - 1] += ` STRAPI_PUBLICATION_STATE=preview`;
+      } else {
```

Comment:
qq: is this to preview staging jobs with the client side build for openapi?

Reply:
Yup, the build commands here are for staging jobs. Although the `oas-page-build` command here would be to also build things statically, as opposed to client-side. The only page to be built client-side should be the preview page for `docs-landing`